hugo/layouts/people/single.html

```diff
@@ -4,6 +4,15 @@
   <h2 id="title">
     <!-- {{- .Title -}} -->
     <span class="font-weight-normal">{{ $person.first }}</span> <span class="font-weight-bold">{{ $person.last }}</span>
+    {{ with $person.orcid }}
+      <a href="https://orcid.org/{{.}}" title="ORCID profile" target="_blank" rel="noopener">
+        <i class="fab fa-orcid fa-sm"></i>
+      </a>
+    {{ else }}
+      <a href="https://aclanthology.org/info/verification" title="How to verify an author" target="_blank" rel="noopener">
```

Comment:
The URL shouldn't be hardcoded. I think this should be `{{ ref . "/info/verification" }}` but haven't tested yet.